src/views/HomeView.vue

```diff
@@ -5,8 +5,10 @@
     import { onMounted } from 'vue';
     
     const store = commonStore();
-    onMounted(() => {
-        store.fetchSubjects()
+    onMounted(async () => {
```

Comment:
If not use this at homepage please remove it

Reply:
home page use it too. filter post/teacher need subjects